roles/cifmw_cephadm/tasks/post.yml

```diff
@@ -46,8 +46,14 @@
       loop: "{{ cifmw_cephadm_log_commands }}"
 
 - name: Configure ceph object store to use external ceph object gateway
+  when:
+    - cifmw_cephadm_vip is defined
+    - cifmw_cephadm_vip default("") | length > 0
```

Comment:
`cifmw_cephadm_vip | default("") | length > 0`

Reply:
Thank you! Fixed